conntrack_linux.go

```diff
@@ -326,11 +326,11 @@ func parseRawData(data []byte) *ConntrackFlow {
 type ConntrackFilterType uint8
 
 const (
-	ConntrackOrigSrcIP = iota // -orig-src ip   Source address from original direction
-	ConntrackOrigDstIP        // -orig-dst ip   Destination address from original direction
-	ConntrackNatSrcIP         // -src-nat ip    Source NAT ip
-	ConntrackNatDstIP         // -dst-nat ip    Destination NAT ip
-	ConntrackNatAnyIP         // -any-nat ip    Source or destination NAT ip
+	ConntrackOrigSrcIP = iota // -orig-src ip    Source address from original direction
+	ConntrackOrigDstIP        // -orig-dst ip    Destination address from original direction
+	ConntrackNatSrcIP         // --reply-src ip  Reply Source IP
+	ConntrackNatDstIP         // --reply-dst ip  Reply Destination IP
+	ConntrackNatAnyIP         // Match source or destination reply IP
```

Comment:
This one still seems to be confusing in this form. It's also not consistent with comments in `MatchConntrackFlow` at the bottom of file.

Reply:
Sure will update the MatchConntrack flow comment, what else is it confusing?

Reply:
I was confused by this "reply" addition but that was my fault in understanding the flow. It's perfectly fine as is.